plugins/org.genivi.faracon/src/org/genivi/faracon/franca2ara/ARAPrimitveTypesCreator.xtend

```diff
@@ -62,4 +66,36 @@ class ARAPrimitveTypesCreator extends Franca2ARABase {
 		return lookupImplementationDataType !== null && lookupImplementationDataType === implementationDataType
 	}
 
+
+	var AUTOSAR primitiveTypesAnonymousArraysModel
+	var ARPackage primitiveTypesAnonymousArraysMainPackage
+
+	def clearPrimitiveTypesAnonymousArrays() {
+		primitiveTypesAnonymousArraysModel = null
+		primitiveTypesAnonymousArraysMainPackage = null
+	}
+
+	def createPrimitiveTypesAnonymousArraysModel() {
```

Comment:
I guess this will have a "merge" conflict with the ARA-Standard-types extension mechanism.
Even if it should be possible to merge, we have the issue that the package names are not guaranteed using the extension mecahnism...

Reply:
As far as I see, it should work even together with a customized set of ARA standard types. The array types created here refer to primitive types that are requested with 'ARAPrimitveTypesCreator.getBaseTypeForReference'. I expect this method to work properly in the customization case as well!?
The only real problem that I see here is that we currently put these fixed sized array types into the package "/ara/stdtypes" which might differ from the package of the custom primitive types. The fixed sized array types should adopt the package of the primitive types. I will fix this after the merge to avoid additional confusions during the merge.